packages/datepicker/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fiscozen/datepicker",
+  "version": "0.1.0",
+  "description": "Design System Datepicker component",
+  "scripts": {
+    "coverage": "vitest run --coverage",
+    "format": "prettier --write src/",
+    "test:unit": "vitest run",
+    "build": "vue-tsc --noEmit && vite build"
+  },
+  "main": "src/index.ts",
+  "type": "module",
+  "keywords": [],
+  "author": "Riccardo Agnoletto",
+  "peerDependencies": {
+    "tailwindcss": "^3.4.1",
+    "vue": "^3.4.13"
+  },
+  "devDependencies": {
+    "@fiscozen/eslint-config": "workspace:^",
+    "@fiscozen/prettier-config": "workspace:^",
+    "@fiscozen/tsconfig": "workspace:^",
+    "@rushstack/eslint-patch": "^1.3.3",
+    "@types/jsdom": "^21.1.6",
+    "@types/node": "^18.19.3",
+    "@vitejs/plugin-vue": "^4.5.2",
+    "@vitest/coverage-v8": "^1.2.1",
+    "@vue/test-utils": "^2.4.3",
+    "@vue/tsconfig": "^0.5.0",
+    "eslint": "^8.49.0",
+    "jsdom": "^23.0.1",
+    "prettier": "^3.0.3",
+    "typescript": "~5.3.0",
+    "vite": "^5.0.10",
+    "vite-plugin-dts": "^3.8.3",
+    "vitest": "^1.2.0",
+    "vue-tsc": "^1.8.25"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@fiscozen/composables": "workspace:^",
+    "@fiscozen/icons": "^0.1.4",
```

Comment:
did you fix the version on purpose?

Reply:
I did, but maybe not the best idea since it already screwed up chromatic builds